smartsim/database/orchestrator.py

```diff
@@ -818,14 +817,18 @@ def _initialize_entities_mpmd(
             run_settings = self._build_run_settings(
                 sys.executable, exe_args_mpmd, db_nodes=db_nodes, port=port, **kwargs
             )
-            output_files = [self.name + ".out"]
-
+            output_files = [self.name + "_0.out"]
         if not run_settings:
             raise ValueError(f"Could not build run settings for {self.launcher}")
-
-        node = DBNode(self.name, self.path, run_settings, [port], output_files)
+        node = DBNode(
+            self.name + "_0",
+            self.path,
+            run_settings,
+            [port],
+            output_files,
+            db_identifier=self.db_identifier,
```

Comment:
I think the issue is that the nodes have no pointer to the Orchestrator, but they are, in some cases, the tasks which are tracked by JM and TM. So I guess we need it, but I can be convinced otherwise.

Reply:
Hmm, after re-looking over the PR I can't think of an immediately trivial way to be able to remove the attr w/o defaulting to a back ref. That would keep the attrs "in sync", but I feel that may be a bigger change than I'm anticipating.

Let's kick this conversation out into a dedicated DB refactor ticket!